Makefile

```diff
@@ -8,9 +8,12 @@ NEW_VERSION ?= $(MAJOR_VERSION).$(MINOR_VERSION).$(shell echo $$(( $(PATCH_VERSI
 ACT ?= go run main.go
 export GITHUB_TOKEN = $(shell cat ~/.config/github/token)
 
-build: 
+build: pkg/runner/resources.go
 	go build -ldflags "-X main.version=$(VERSION)" -o dist/local/act main.go
 
+pkg/runner/resources.go:
+	go-bindata -pkg runner -o pkg/runner/resources.go -prefix pkg/runner/res/ pkg/runner/res/
```

Comment:
go-bindata is archived.  check out https://github.com/gobuffalo/packr

Reply:
https://github.com/gobuffalo/packr:
> NOTICE: Please consider migrating your projects to github.com/markbates/pkger. It has an idiomatic API, minimal dependencies, a stronger test suite (tested directly against the std lib counterparts), transparent tooling, and more.

It looks like https://github.com/markbates/pkger is the current thing?

It's pretty frustrating implementing something, being asked to reimplement, reimplementing, and being sent to reimplement again using something which is also obsolete.

I'm not sure when I'll be able to work on this. We are working on runners today, and I have some changes to my own action which may require me to work on nektos to make sure it remains supported, but... I'm not at all certain about timelines...

Reply:
go 1.16 introduced go:embed